3rdparty/fastcv/hal/fastcv_hal_core.cpp

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "fastcv_hal_core.hpp"
+#include "fastcv_hal_utils.hpp"
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/base.hpp>
+#include <opencv2/core/utils/logger.hpp>
+
+bool isInitialized = false;
```

Comment:
This can be moved to `fastcv_hal_utils.cpp`

Reply:
will update in next PR